bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/AdminStyleForm.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSelect } from './StyleSelect';
+import { Controller } from 'oskari-ui/util';
+
+export const AdminStyleForm = (props) => {
+    return (
+        <StyleSelect layer={ props.layer } controller={ props.controller } />
+    );
+};
+
+AdminStyleForm.propTypes = {
+    oskariStyle: PropTypes.object,
```

Comment:
The prop `oskariStyle` is not used and I don't see how it would be used here. Remove it from propTypes.